electron/preload.js

```diff
@@ -4,10 +4,10 @@ const { fileURLToPath } = require('url')
 const { channels } = require('../src/shared/constants')
 
 const { importFolder } = require('./lib/import-folder')
-const { getDiograph } = require('./lib/get-diograph')
 const { getDioryFolderLocation } = require('./lib/get-diory-folder-location')
 const { setDioryFolderLocation } = require('./lib/set-diory-folder-location')
-const { saveDiograph } = require('./lib/save-diograph')
+
+require('./folder-connector')
```

Comment:
Electron 28.0.0 finally supports ES modules: https://www.electronjs.org/docs/latest/tutorial/esm

Need to upgrade to that at some point!